src/content/stackoverflow/range.test.js

```diff
@@ -40,7 +39,7 @@ describe('range', () => {
     const { start, end } = getTextPosition(nodeWithTwoNodes, expectedText);
 
     const range = getRangeOfPositions(nodeWithTwoNodes, start, end);
-
+    document.write("hello world");
```

Comment:
Don't forget to delete it

Reply:
Thanks, that was a test I didn't delete before committing.